samples/buildstrategy/buildkit/buildstrategy_buildkit_cr.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: shipwright.io/v1alpha1
+kind: ClusterBuildStrategy
+metadata:
+  name: buildkit
+  annotations:
+    # See https://github.com/moby/buildkit/blob/master/docs/rootless.md#about---oci-worker-no-process-sandbox for more information
+    container.apparmor.security.beta.kubernetes.io/step-build-and-push: unconfined
+    container.seccomp.security.alpha.kubernetes.io/step-build-and-push: unconfined
+spec:
+  buildSteps:
+    - name: prepare
+      image: alpine:latest
```

Comment:
To reduce the number of images in a build strategy, one can try to use the main image here. I assume `moby/buildkit:master-rootless` contains a shell. Did you try that?

Reply:
Not possible, check the [entrypoint](https://github.com/moby/buildkit/blob/master/Dockerfile#L310), it runs `buildkitd` binary.

Reply:
I do like the separation of concerns, it make it obvious . I think we will get rid of this once we move to our new Shipwright custom resources.